kernel/include/devices/LApic.h

```diff
@@ -111,6 +113,8 @@ namespace Kernel::Devices
         uint64_t timerTicksPerMs;
         uint32_t calibratedDivisor;
```

Comment:
Unrelated, but while you're touching apic code: could you move calibratedDivisor up to the line above timerTicksPerMs.
It'll just save 8 bytes per instance of the APIC class, now that I see it haha.